services/WAF/test_script/rate-attacks.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+token="eyJraWQiOiJsRVE1N29YdW9EYlhcL0ExK2tRa3RydEtQNzVrenJPeFZEK1liXC9ZZTNtUEk9IiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiI0OWM4ZmFmYy1mY2Q0LTQxZjEtODE3Zi1kNTc1OWUwMzg0ZjciLCJhdWQiOiIyNmc1NWczaWtkbHRkYmJhMzhicTk1Z2UxcyIsImVtYWlsX3ZlcmlmaWVkIjp0cnVlLCJldmVudF9pZCI6IjMxOTllOTk5LTdkMjktNGM5NS05N2E2LTZiMDA1MmE1MjJhMCIsInRva2VuX3VzZSI6ImlkIiwiYXV0aF90aW1lIjoxNjAyNTY3NjQzLCJpc3MiOiJodHRwczpcL1wvY29nbml0by1pZHAudXMtZWFzdC0xLmFtYXpvbmF3cy5jb21cL3VzLWVhc3QtMV9MWHhTd1EwdUIiLCJjb2duaXRvOnVzZXJuYW1lIjoia2l0Y2h1NjY2QGdtYWlsLmNvbSIsImV4cCI6MTYwMjU3MTI0MywiaWF0IjoxNjAyNTY3NjQzLCJlbWFpbCI6ImtpdGNodTY2NkBnbWFpbC5jb20ifQ.SB3NlwsCZXO0Bnl6zCAFWUi38FLbYXzxQ8u3qncTZS-vB22Y-6WdsPEFGY8dHNREqGByJq9xbO_g0gfxC-Rs_tkpN-Nhf2DBAxHPwtQhbowDQq4RFZ4sCO3Am-Pva6XXl_r5_4UWRobIfsR3ihmyB-RuTVpnf1o7yuzIRNptmqIt5iluAfmbuocsNoTNrWvkT4su3IQiHIyZQIz3TzX6dIOGzuLD-sD9tjOAXwwaJSCjmZn4fAhe7bko-gOfTcalWkgPTRt7WEl4wlRl-rsrIWtriC8Q3ssw8-VnOP_T3b1XvH2jHS5DsG3TrKkgKdYybw6MfDq72kj0R8yytgOsuw"
```

Comment:
Let's not commit any tokens in the repo. I'd suggest to have a placeholder instead.

```suggestion
token="${TOKEN}"
```